TestCases/rans/naca0012/turb_NACA0012_sst_fixedvalues.cfg

```diff
@@ -0,0 +1,260 @@
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES,
+%                               WAVE_EQUATION, HEAT_EQUATION, FEM_ELASTICITY,
+%                               POISSON_EQUATION)                             
+SOLVER= RANS
+%
+% Specify turbulent model (NONE, SA, SA_NEG, SST)
+KIND_TURB_MODEL= SST
+%
+% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT)
+MATH_PROBLEM= DIRECT
+%
+% Restart solution (NO, YES)
+RESTART_SOL= NO
+%
+% Read binary restart files (YES, NO)
+READ_BINARY_RESTART= NO
+%
+
+% -------------------- COMPRESSIBLE FREE-STREAM DEFINITION --------------------%
+%
+% Mach number (non-dimensional, based on the free-stream values)
+MACH_NUMBER= 0.15
+%
+% Angle of attack (degrees, only for compressible flows)
+AOA= 10.0
+%
+% Free-stream temperature (288.15 K by default)
+FREESTREAM_TEMPERATURE= 300.0
+%
+% Reynolds number (non-dimensional, based on the free-stream values)
+REYNOLDS_NUMBER= 6.0E6
+%
+% Reynolds length (1 m by default)
+REYNOLDS_LENGTH= 1.0
+%
+% Free-stream turbulence intensity
+FREESTREAM_TURBULENCEINTENSITY= 0.0008165
+% 
+% Free-stream ratio between turbulent and laminar viscosity
+FREESTREAM_TURB2LAMVISCRATIO= 1.2
+%
+% Fix turbulence quantities to far-field values at some upstream half-space
+TURB_FIXED_VALUES= YES
+%
+% Shift of the fixed values half-space in unit far-field velocity vectors
+TURB_FIXED_VALUES_DOMAIN= -1.0
+
+% ---------------------- REFERENCE VALUE DEFINITION ---------------------------%
+%
+% Reference origin for moment computation
+REF_ORIGIN_MOMENT_X = 0.25
+REF_ORIGIN_MOMENT_Y = 0.00
+REF_ORIGIN_MOMENT_Z = 0.00
+%
+% Reference length for pitching, rolling, and yawing non-dimensional moment
+REF_LENGTH= 1.0
+%
+% Reference area for force coefficients (0 implies automatic calculation)
+REF_AREA= 1.0
+%
+% Flow non-dimensionalization (DIMENSIONAL, FREESTREAM_PRESS_EQ_ONE,
+%                              FREESTREAM_VEL_EQ_MACH, FREESTREAM_VEL_EQ_ONE)
+REF_DIMENSIONALIZATION= FREESTREAM_PRESS_EQ_ONE
+
+% -------------------- BOUNDARY CONDITION DEFINITION --------------------------%
+%
+% Navier-Stokes wall boundary marker(s) (NONE = no marker)
+MARKER_HEATFLUX= ( airfoil, 0.0 )
+%
+% Farfield boundary marker(s) (NONE = no marker)
+MARKER_FAR= ( farfield )
+%
+% Marker(s) of the surface to be plotted or designed
+MARKER_PLOTTING= ( airfoil )
+%
+% Marker(s) of the surface where the functional (Cd, Cl, etc.) will be evaluated
+MARKER_MONITORING= ( airfoil )
+
+% ------------- COMMON PARAMETERS DEFINING THE NUMERICAL METHOD ---------------%
+%
+% Numerical method for spatial gradients (GREEN_GAUSS, WEIGHTED_LEAST_SQUARES)
+NUM_METHOD_GRAD= WEIGHTED_LEAST_SQUARES
+NUM_METHOD_GRAD_RECON= LEAST_SQUARES
```

Comment:
Wait, I recall there were some accuracy problems with LSQ gradient. Introduced in/ and see the discussion in #790 . Was it fixed in the meantime or were the issues not that bad anyway @pcarruscag ?